SemanticOverrides.md

```diff
@@ -0,0 +1,85 @@
+Semantic Overrides API
+===================
+Allows users and curators to add enrichments, remove enrichments, and make enrichments selected or unselected in real time for 
+any canonical term and any taxonomy supported by Semantic 1.0
+
+#Global Overrides
+Global overrides are administrated by authorized curators via the Overrides UI tool. The tool is located at 
+http://www.careerbuilder.com/nlUtils/SemanticOverridesTool/SemanticOverride.html?g=true
+The tool will only work for those on an approved list. For questions about gaining access, send an email to 
+DataScienceApplicationDevelopment@careerbuilder.com
+
+#User Overrides
+User overrides are only applied to Semantic Search requests with a corresponding user_id, 
+allowing personalization of Semantic Search results. User overrides have priority over global overrides, and are applied to
+any extracted keyword for which an override exists. 
+
+##User Overrides Request
+
+User overrides are added and deleted using a RESTful interface.
+
+###HTTP method PUT
+
+Adds or updates an override for the specified user_id, term, language, taxonomy, and enrichment combination. 
+Since this endpoint modifies override state, it is restricted to certain OAuth client ids. For questions about access, send an email to 
+DataScienceApplicationDevelopment@careerbuilder.com
+
+Parameters (URL-encoded path parameters):
```

Comment:
this API relies heavily on usage of path parameters, which is slightly more complex than the usual URI query params / form urlencoded params most of our APIs use. I think it might be nice to demonstrate the structure here (without example values, just URI template-style). something like [...]/user/{user_id}/{term}/[...]